receiver.html

```diff
@@ -0,0 +1,20 @@
+<!DOCTYPE html>
+<html lang="en">
+<head>
+    <meta charset="UTF-8">
+    <meta name="viewport" content="width=device-width, initial-scale=1.0">
+    <meta http-equiv="X-UA-Compatible" content="ie=edge">
+    <title>Ooyala Receiver</title>
+
+    <script type="text/javascript" src="//www.gstatic.com/cast/sdk/libs/receiver/2.0.0/cast_receiver.js"></script>
+    <script type="text/javascript" src="//player.ooyala.com/core/e48c9b51282d406f957c666149763424?plugins=bm,cast"></script>    
+</head>
+<body>
+    <div id="splash-screen" class="fbox-container fs">
+        <img id="splash-image" src="images/logo.svg" />
+        <div id="status-cast">
+        </div>
+    </div>
+    <div id="player"></div>
+<script type="text/javascript" src="//static.ooyala.com/chromecast/latest/cast.min.js"></script></body>
```

Comment:
This URL is not correct. You have to use
//player.ooyala.com/static/chromecast/latest/cast.min.js

Reply:
In that case I need to ask access to player.ooyala.com/static/ on the s3 buckets